docs/tutorials/custom_contracts/ethereum.md

```diff
@@ -747,6 +811,36 @@ You can see in the event received over the WebSocket connection, the blockchain
 
 In the event, we can also see the `blockchainevent` itself, which has an `output` object. These are the `params` in our FireFly Interface, and the actual output of the event. Here we can see the `value` is `3` which is what we set the integer to in our original transaction.
 
+### Subscription offset
+If you query by the ID of your subscription with the `fetchstatus` parameter, you can see its current `offset`.
+
+`GET` `http://localhost:5000/api/v1/namespaces/default/subscriptions/f826269c-65ed-4634-b24c-4f399ec53a32`
+```json
+{
+  "id": "f826269c-65ed-4634-b24c-4f399ec53a32",
+  "namespace": "default",
+  "name": "simple-storage",
+  "transport": "websockets",
+  "filter": {
+    "events": "blockchain_event_received",
+    "message": {},
+    "transaction": {},
+    "blockchainevent": {
+      "listener": "1bfa3b0f-3d90-403e-94a4-af978d8c5b14"
+    }
+  },
+  "options": {
+    "firstEvent": "-1",
```

Comment:
I don't doubt this is actually what the API said the first event was, but I'm wondering if that is what the API was supposed to say here. `-1` doesn't make much sense to me. `0` does. But not `-1`.